index.bs

```diff
@@ -195,6 +195,19 @@ the user-agent should retain the previously computed layout state if possible.
   values are added on top of any existing containment values.
 </div>
 
+Restrictions and Clarifications {#restrictions}
+===============================
+
+* In situations where [=layout containment=] has no effect (e.g. the
+    element does not generate a principal box), 'subtree-visibility' values also
+    have no effect. In other words, if [=layout containment=] has no effect, the
+    'subtree-visibility' value is effectively [=visible=].
+
+* <a href="https://html.spec.whatwg.org/multipage/rendering.html#replaced-elements">
```

Comment:
What about:

Object, embed, iframe?

Reply:
We also don't paint iframe contents (we have tests for that). I'll add tests for object and embed, but I think we should not separate things into different subgroups of replaced if we don't have a reason to do so. So, I think this phrasing _should_ be correct. WDYT?

Reply:
Ok, but then you should either omit "such as images" to avoid it sounding like it's just for images.